CHANGELOG.md

```diff
@@ -3,7 +3,163 @@
+- topdown/units_parse: Avoid extra decimal places for integers. (#5011)
+- Add error type to units.* builtin test assertions (#4966)
+- test/e2e/certrefresh: Add file.Sync() to eliminate test failures due to slow disk writes. (#5033)
+- testcases: Fix duplicate yaml test keys (#4970)
+- topdown/exported_tests: Remove golang 1.16 x509 exception (#5032)
+- wasm/tests/test: Fix broken WASM C tests for is_type builtins. (#5075)
+- cmd/bench: Fix port collision in utility function used for E2E testing.
+
+### Documentation
+
+- SECURITY: Migrate policy to web site, update content (#5094) ([#4272](https://github.com/open-policy-agent/opa/issues/4272)) reported by @adoliver
+- Add deprecated flag to all deprecated builtins (#5078) ([#5072](https://github.com/open-policy-agent/opa/issues/5072))
+- builtins: Update description of format_int to say it rounds down
+- docs/policy-reference: Update Rego EBNF grammar (#4955) (authored by @shaded-enmity)
+- docs/builtins: Fix typo in semver.compare (#5059) ([#5012](https://github.com/open-policy-agent/opa/issues/5012)) reported by @tetsuya28
+- docs: Fix AWS Signature section in Configuration (authored by @pauly4it)
+- docs: Update port and bundle folder for GraphQL tutorial
+- docs: Document that function overloading is unsupported
+- docs: Fixing related_resources annotations example ([#4982](https://github.com/open-policy-agent/opa/issues/4982)) reported by @humbertoc-silva
+- docs: Fixing typo in metadata, issue #5018 (#5019) ([#5018](https://github.com/open-policy-agent/opa/issues/5018)) authored by @cimin0 reported by @cimin0
+
+### Website + Ecosystem
+
+- Update links to opa-kafka-plugin (#5021)
+- Add OCI documentation (#4781) (authored by @carabasdaniel)
+- Fix broken link (#5071) (authored by @unautre)
+- Add article on using OPA for data filtering in Kafka (#5082)
+- Ecosystem: add some links to Rönd (#5102) (authored by @ugho16)
+- Add community integration (fiber) (#5026) (authored by @mstrYoda)
+- Add Spacelift Integration (authored by @theseanodell)
+
+- Ecosystem Additions:
+  - cosign (#5040) (authored by @Dentrax)
+
+### Miscellaneous
+
+- Add VNG Cloud to adopters (#5001) (authored by @vinhph0906)
+
+- Dependency bumps, notably:
+  - build: bump golang: 1.19 -> 1.19.1
+  - build: use go 1.19, drop go 1.16
+  - build(deps): bump aquasecurity/trivy-action from 0.6.1 -> 0.7.1
+  - build(deps): bump github.com/agnivade/levenshtein from 1.0.1 -> 1.1.1
+  - build(deps): bump github.com/containerd/containerd from 1.6.6 -> 1.6.8
+  - build(deps): bump github.com/go-ini/ini from 1.66.6 -> 1.67.0
+  - build(deps): bump github.com/prometheus/client_golang
+  - build(deps): bump google.golang.org/grpc from 1.48.0 -> 1.49.0
+  - build(deps): bump tj-actions/changed-files from 28.0.0 -> 29.0.3
+
+- Dependency removals:
+  - internal: Vendor gqlparser library (#5067) ([#5065](https://github.com/open-policy-agent/opa/issues/5065)) authored by @philipaconrad reported by @vikstrous2
+
+### Miscellaneous
```

Comment:
Multiple `Misc` sections

Reply:
Ah, nuts. A leftover from the changelog generator script. I'll patch that out shortly. Thanks for catching that! :+1:

Reply:
Rolled this into the latest version.